.github/workflows/publish-deb.yml

```diff
@@ -26,7 +23,6 @@ jobs:
       - name: Init basic publish env
         run: |
           sudo apt-get update
-          sudo apt install -y gpg ca-certificates
```

Comment:
remove `gpg` means that we need to sign the generated deb artifacts manually after we get them?

Reply:
Just upload to CI Action, no signing required. The debian repo currently needs to be built locally and uploaded to cos.